projects/common/lib/components/replace-link-pipe/i18n/data/en/index.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
```

Comment:
I think we can remove this pipe and simplify things. Look at how we stored all the links in FPC.

https://github.com/bcgov/moh-fpcare/blob/master/src/environments/environment.ts#L39-L44

All we had to do then was as simple as:

https://github.com/bcgov/moh-fpcare/blob/master/src/app/modules/core/components/consent-modal/consent-modal.component.html#L23


Note - this does require that links are stored in the application - not the library - which is desired behaviour. 

As such, I think we can remove this pipe entirely. I misunderstood you when we talked last time, I thought this was a pipe to parse links at runtime in JSON responses from the server, not to use hardcoded links in the application.